README.md

```
@@ -108,10 +109,12 @@ jobs:
 | `pr`               | Whether to create a pull request with version changes         | No       | `false`                 |
 | `pr_auto_merge`    | Whether to automatically merge the PR if all checks pass      | No       | `false`                 |
 | `pr_message`       | Message to use for the pull request                           | No       | `chore: version update` |
+| `pr_version_prefix` | Prefix for version PR branch names (e.g., "feature/" creates "feature/version-1.2.3") | `version_bump` |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)